website/content/docs/platform/k8s/vso/sources/vault/index.mdx

```diff
@@ -112,6 +113,72 @@ spec:
   # headers: []
 ```
 
+### VaultAuthGlobal custom resource
+
+<Tip title={"Feature availability"}>
+
+VSO v0.8.0
+
+</Tip>
+
+Namespaced resource that provides shared Vault authentication configuration that can be inherited by multiple
+`VaultAuth` custom resources. It supports multiple authentication methods and allows you to define a default
+authentication method that can be overridden by individual VaultAuth custom resources. See `vaultAuthGlobalRef` in
+the [VaultAuth spec][va-spec] for more details. The `VaultAuthGlobal` custom resource is optional and can be used to
+simplify the configuration of multiple VaultAuth custom resources by reducing config duplication. Like other
+namespaced VSO custom resources, there can be many VaultAuthGlobal resources configured in a single Kubernetes cluster.
```

Comment:
```suggestion
namespaced VSO custom resources, there can be many `VaultAuthGlobal` resources configured in a single Kubernetes cluster.
```

Should we have custom resource kinds inside `` consistently?

Reply:
We only do that for the first one, otherwise they can overwhelm the reader.